test/unit.js

```diff
@@ -13,7 +13,23 @@ const mocha = new Mocha({
   grep: argv.grep ? argv.grep : undefined,
 });
 
-Glob(`src/**/${argv.component ? argv.component : '*'}.spec.js`, {}, (err, files) => {
+const paths = [
+  `src/**/*${argv.component ? argv.component : ''}*.spec.js`,
```

Comment:
Can we put the `*` back inside the ternary on this line? I moved it in a PR so that `-c Step` for example wouldn't run tests for every component beginning with "Step" (StepButton, StepLabel, Stepper, StepContent, StepConnector, etc...)